inc/class-shortcode-ui.php

```diff
@@ -191,23 +190,33 @@ private function compare_shortcodes_by_label( $a, $b ) {
 	public function handle_ajax_do_shortcode() {
 
 		// Don't sanitize shortcodes — can contain HTML kses doesn't allow (e.g. sourcecode shortcode)
-		$shortcode = ! empty( $_POST['shortcode'] ) ? stripslashes( $_POST['shortcode'] ) : null;
-		$post_id   = ! empty( $_POST['post_id'] ) ? intval( $_POST['post_id'] ) : null;
+		if ( ! empty( $_POST['shortcode'] ) ) {
+			$shortcode = stripslashes( $_POST['shortcode'] ); // WPCS: input var okay
```

Comment:
`stripslashes()` isn't a santizing function. I don't know what `$_POST['shortcode']` might contain, but you should put it through a sanitization function if possible. If it is allowed to contain HTML then consider putting it through `wp_kses()`. If it is OK for it to contain scripts, then you'll just need to add "sanitization" to the WPCS whitelising comment.